### PR TITLE
Feature update dependencies

### PR DESCRIPTION
**Pull request by JosefBredereck**
Summary of changes:
Updated dependencies of different packages while they have npm audits for lodash, tap and even a critical with the handlebars package.

### CODE REVIEW

**Comment by ringods** [comment]
@sghoweri or @bmuenzenmeyer is there anyone else who could process this PR and create a new release of the packages?

**Comment by bmuenzenmeyer** [comment]
i can test this tomorrow

File: packages/live-server/package.json [hunk 1]
```diff
@@ -12,14 +12,14 @@
   ],
   "author": "Tapio Vierros",
   "dependencies": {
-    "chokidar": "1.6.0",
+    "chokidar": "3.1.0",
```
Comment (bmuenzenmeyer) [inline packages/live-server/package.json]: why upgrade this here but not in core? I am a big fan of using tools like https://www.npmjs.com/package/npm-check-updates to perform actions like this across the monorepo

**Comment by bmuenzenmeyer** [comment]
tested this out - thanks for taking the time to improve the codebase. Will uptick that chokidar dependency while I am here

**Comment by bmuenzenmeyer** [comment]
updating chokidar in core seems to have broken live-reload. confirming

File: packages/live-server/package.json (hunk 1 above)
Comment (JosefBredereck) [reply packages/live-server/package.json]: Hey @bmuenzenmeyer I walked my way through the directories and it seems like I missed the core with chokidar. Thanks for the note and providing a link to check the monorepro a better way.

Comment (bmuenzenmeyer) [reply packages/live-server/package.json]: no problem! turns out it was good to omit for now anyways, see #1069 😊

**Pull request merged**